utilities/shared.mjs

```diff
@@ -32,6 +32,18 @@ export function validateProjectID(id){
    return false
 }
 
+export function validateLineID(id) {
```

Comment:
I think all functions will share a `validateID()` function.  This does the same thing as the validateProjectID() function, so we won't need it.  

I will clean this upstream after we merge.

Reply:
Can we use the ProjectID here @thehabes ?

Reply:
I agree with you and updated the same in the code .